example/test-jvm.sh failed which sounds suspicious because it tests IOApp termination and the PR touches the same functionality, but consider this output:
```
[info] running cats.effect.example.Example 
started
Timed out waiting for 'started' in output
Error: Process completed with exit code 255.
```
Also, running `example/test-jvm.sh 2.13.3` locally failed for me once, on the first run where some compilation was performed with the same message as above and passed on 5 or so subsequent runs.
In conclusion I believe this test is flaky because 30s timeout is too tight.

--- a/example/test-jvm.sh
+++ b/example/test-jvm.sh
@@ -16,7 +16,7 @@ await-output() {
     c=$(($c+1))
   done
 
-  if [[ $c -gt 20 ]]; then
+  if [[ $c -gt 60 ]]; then
     echo -e "\e[31mTimed out waiting for '$1' in output\e[0m"
     return -1
   else

